src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -142,13 +142,13 @@ const ProjectInstallPlatform = React.createClass({
             <div dangerouslySetInnerHTML={{__html: this.state.html}}/>
           )}
 
-          {queryParams.onboarding ?
+          {'signup' in queryParams ?
```

Comment:
Oh, in that case, do `queryParams.hasOwnProperty('signup')`. It's more reliable than `'signup' in queryParams` for JavaScript.